docs/Task-Deployment.md

```diff
@@ -22,7 +22,7 @@ Whether running via the inividual CFn templates or the "parent" template, the ge
     1. _Database:_ To be applied to the stack's ASG-managed EC2 instance and to its RDS service. Allows mysql traffic over standard 3306/TCP between the EC2 instance and its RDS database. All other sources/destinations are blocked.
 1. _Execute the EFS service setup:_ A generic, three-AZ EFS share is configured. The "storage" security group is applied to allow NFS traffic only from the RedMine EC2 host.
 1. _Execute the ELB service setup:_ An application load-balancer is configured. The "application" security group is applied to all HTTPS-based requests from any source and allow HTTP-based communications only between the active RedMine EC2 instance and the load-balancer. A user-selected, ACM-managed SSL certificate is bound to the load-balancer.
-1. _Execute the IAM role setup:_ An IAM policy and attachable instance-role is created. The IAM objects allow unfettered access to the to the S3 bucket from the RedMine-hosting EC2 instances for the purposes of attachment backups. The IAM policy also allows: the AutoScaling service to launch and terminate EC2 instances the the ASG-managed EC2 instaces to use CloudWatch agent for log-export/centralized-logging; and, control of the EC2s via AWS Systems Manager.
+1. _Execute the IAM role setup:_ An IAM policy and attachable instance-role is created. The IAM objects allow unfettered access to the to the S3 bucket from the RedMine-hosting EC2 instances for the purposes of attachment backups. The IAM policy also allows the AutoScaling service to launch and terminate EC2 instances the the ASG-managed EC2 instaces to use CloudWatch agent for log-export/centralized-logging; and, control of the EC2s via AWS Systems Manager.
```

Comment:
I'm still struggling with this sentence:

> The IAM policy also allows the AutoScaling service to launch and terminate EC2 instances the the ASG-managed EC2 instaces to use CloudWatch agent for log-export/centralized-logging; and, control of the EC2s via AWS Systems Manager.

Reply:
My keyboard's been wonky, lately (at over 2.5 years old, it's lasted longer than any laptop keyboard I've ever had). I think something got zapped.